README.md

```diff
@@ -5,111 +5,105 @@
-The reason is that it is much harder to use beacon tokens when the address is made up of completely unique credentials. The beacon tokens should only be usable with addresses of the DEX. If their is no way to clearly distinguish a DEX's address from any other address, guaranteeing proper usage of the beacon tokens is virtually impossible. v1.0.0 of Cardano-Swaps used unique spending scripts; you can read about the limitations in the v1.0.0 commit README.
+## Specification
 
-### Delegating
-With each user having their own DEX address, delegation the address is identical to delegating a traditional payment address that uses either a staking pubkey or a staking script. You can check out [delegation section](GettingStarted.md#delegate-the-swap-address) of GettingStarted.md for an example using a staking pubkey.
+### Personal Contracts
+Cardano addresses are made up of both a payment credential and a staking credential. As long as the staking credential is unique to that user, delegation control over the address is maintained. Cardano-swaps leverages this duality by giving users addresses that are composed of the same spending scripts (per swap pair) and unique staking credentials. The spending credential is implemented in a way that gives the staking credential authority over all owner related actions, besides the actual swap execution. 
 
----
-## Beacon Tokens
-Upon reading how all users have their own DEX addresses, it is natural to ask how potential "swappers" can find all available swaps and the information needed to remotely swap with them. The answer is to use beacon tokens.
+:note: To force the use of a staking credential, it is not possible to mint a beacon token to an address without a staking credential.
 
-Using [Koios](https://api.koios.rest/#overview) or [Blockfrost](https://docs.blockfrost.io/) apis, it is possible to find all addresses that contain a specific native token. Once you have the address, you can then use Koios or Blockfrost again to get all utxos locked at those addresses.
+### Minting and Using Beacon Tokens
+Beacon Tokens are used to tag `cardano-swaps` addresses so they are readily queryable via an off-chain API, such as Koios or Blockfrost. It is relatively straightforward to find all addresses that contain a specific native token. Here are some examples:
 
 | Task | Koios Api | Blockfrost Api |
 |--|--|--|
 | Addresses with a beacon | [api](https://api.koios.rest/#get-/asset_address_list) | [api](https://docs.blockfrost.io/#tag/Cardano-Assets/paths/~1assets~1%7Basset%7D~1addresses/get)|
 | UTxOs at the address | [api](https://api.koios.rest/#post-/address_info) | [api](https://docs.blockfrost.io/#tag/Cardano-Addresses/paths/~1addresses~1%7Baddress%7D~1utxos/get)|
 
-The utxos api also returns which utxos contain reference scripts. This is how users can remotely execute swap contracts.
+The UTxO query also returns which UTxOs contain reference scripts, allowing users execute their swaps.
```

Comment:
allowing users *to remotely* execute their swaps

Reply:
wdym remotely? not clear

Reply:
My thought is that the swapper is referencing the script from a far but I guess it isn't necessary. Just add the *to* for grammar.